R/ensure-tpch-source.R

```diff
@@ -4,36 +4,67 @@
 #' @export
 tpch_tables <- c("customer", "lineitem", "nation", "orders", "part", "partsupp", "region", "supplier")
 
+datalogistik_available <- function() {
+  exit_code <- system("which datalogistik")
+
+  if (exit_code != 0) {
+    warning(
+      'datalogistik not installed or on $PATH. If installed with pipx, ensure it ',
+      'is on $PATH, e.g. by adding PATH="${PATH}:${HOME}/.local/bin" to ~/.Renviron'
+    )
+  }
+
+  exit_code == 0
+}
```

Comment:
I can work on something that does this if you're busy with other things, but ideally we would have a helper function that does the installation + helpful messages | docs if that doesn't work or `pipx` isn't installed, etc.